doc/library/gof/wrapper.txt

```diff
@@ -0,0 +1,15 @@
+.. _libdoc_gof_wrapper:
+
+=============================================
+:mod:`wrapper` -- Wrapper class for op params
```

Comment:
I have copied the syntax from `doc/library/gof.type.txt` and from the other files in this folder, in which `:mod:` is always associated to a short name, not a complete name (e.g. from `type.txt`: `:mod:`type` -- Interface for types of variables`).

Should I write `theano.gof.wrapper` anyway ?